Task03Trie/Task03Tests/TrieTests.cs

```diff
@@ -0,0 +1,58 @@
+using Task03Trie;
+namespace Task03Tests;
```

Comment:
```suggestion
namespace Task03Tests;

using Task03Trie;
```
namespace пишется первой строкой в файле после шапки с лицензией